integrations/access/accessrequest/app.go

```diff
@@ -122,14 +122,22 @@ func (a *App) Err() error {
 
 func (a *App) run(ctx context.Context) error {
 	process := lib.MustGetProcess(ctx)
+	ctx, log := logger.WithField(ctx, "plugin", a.pluginName)
+
+	watchKinds := []types.WatchKind{
+		{Kind: types.KindAccessRequest},
+	}
+
+	if err := a.initAccessMonitoringRulesCache(ctx); err != nil {
+		log.WithError(err).Errorf("initialising Access Monitoring Rule cache")
+	} else {
+		watchKinds = append(watchKinds, types.WatchKind{Kind: types.KindAccessMonitoringRule})
+	}
 
 	job, err := watcherjob.NewJob(
 		a.apiClient,
 		watcherjob.Config{
-			Watch: types.Watch{Kinds: []types.WatchKind{
-				{Kind: types.KindAccessRequest},
-				{Kind: types.KindAccessMonitoringRule},
-			}},
+			Watch:            types.Watch{Kinds: watchKinds},
```

Comment:
If a rule is created between this `initAccessMonitoringRulesCache` and starting to watch events, we'll not load it into the cache.

I think we can keep things as is and just set the watcher config option `AllowPartialSuccess` to true:
```suggestion
			Watch:            types.Watch{Kinds: watchKinds, AllowPartialSuccess: true},
```

Docs:
https://github.com/gravitational/teleport/blob/2407f27cff24a54041436c4bbe981ab79448e4d6/api/types/events.go#L101-L107